content/docs/product-tour.mdx

```diff
@@ -14,94 +14,94 @@ Here we give a quick **2 minute overview of TinaCMS**, and how it fits into your
 
 TinaCMS is an open-source Content Management System (CMS) that seamlessly integrates with your Markdown workflow.
 
-![Intro-gif](https://res.cloudinary.com/forestry-demo/image/upload/v1645712511/tina-io/docs/your-blocks.gif)
+<WebmEmbed embedSrc="/img/docs/your-blocks.webm"/>
 
 ### Your Website
 
 - With TinaCMS, The developer hosts the site where they like, using whichever framework they choose.
 
-![Website-png](https://res.cloudinary.com/forestry-demo/image/upload/v1689874025/tina-io/docs/product-tour/headless_cms1.png)
+![Website-png](/img/docs/product-tour/headless_cms1.png)
 
 ### Headless CMS
 
 - TinaCMS is "Headless", meaning it's decoupled from your website. TinaCMS provides a queryable content API that your website's code consumes.
 
-![Headless-png](https://res.cloudinary.com/forestry-demo/image/upload/v1689874025/tina-io/docs/product-tour/headless_cms2.png)
+![Headless-png](/img/docs/product-tour/headless_cms2.png)
 
 ## The CMS Backend
 
 - The TinaCMS backend provides the API for querying your content, offering support for filtering, searching, and pagination.
 
-![Queryable-db-png](https://res.cloudinary.com/forestry-demo/image/upload/v1689874024/tina-io/docs/product-tour/queryable_database.png)
+![Queryable-db-png](/img/docs/product-tour/queryable_database.png)
 
 ### Git-backed Content
 
 - All your content gets backed by git into JSON/Markdown/MDX files
 
-![Git-backed-png](https://res.cloudinary.com/forestry-demo/image/upload/v1689874025/tina-io/docs/product-tour/git-backed.png)
+![Git-backed-png](public/img/docs/product-tour/git-backed.png)
```

Comment:
**issue**: Image not loaded due to "public" filename prefix.

Reply:
Resolved